### DISCUSSION TITLE
Migrating from standalone Z2M (pi4) to Z2M on Home Assistant OS (NUC) - backup and restore process?

### DISCUSSION THREAD

**Discussion by rocket59** [Koenkk/zigbee2mqtt#12836] [Devices]
Sorry for the long title. 

As it says I'm moving my z2m setup (including my conbee 2 coordinator). Is there a process for transferring my devices to the new setup in Home Assistant (this will be the full Home Assistant OS installed on an intel NUC) without re-pairing? What should I be copying across?

I did a search on the discussions and unfortunately am still unclear as to how to do this. Thanks for any advice!

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/guide/faq/#how-do-i-move-my-zigbee2mqtt-instance-to-a-different-environment

**Comment by rocket59** [comment]
Thank you! I missed that.
